oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -103,6 +103,8 @@ receiver-jvm:
   default:
 receiver-clr:
   default:
+receiver-so11y:
+  default:
```

Comment:
This receiver should be closed, due to `telemetry/so11y` closed.

Reply:
done